packages/arb-token-bridge-ui/tests/e2e/specs/depositCctp.cy.ts

```diff
@@ -112,12 +112,11 @@ describe('Deposit USDC through CCTP', () => {
             // eslint-disable-next-line
             cy.wait(40_000)
             cy.confirmMetamaskTransaction().then(() => {
-              cy.findByText('Pending transactions').should('be.visible') // tx history should be opened
```

Comment:
I would leave out utils for opening/closing transaction history in another util, so they are more composable

think about this use case for example:

```
cy.openTransactionHistory('pending')
cy.findTransaction('0.124 ETH')


cy.openTransactionHistory('settled')
// assert that there is none
```

Reply:
This will be moved to [selectTransactionsHistoryTab](https://github.com/OffchainLabs/arbitrum-token-bridge/pull/1809)

Reply:
We have 2 commands for that now:
- `selectTransactionsPanelTab(tab)`: select the proper tab once transactions panel is opened
- `openTransactionsPanel(tab)`: open the transaction panels and select a tab (reusing `selectTransactionsPanelTab`)

In few cases (after a transaction is done for example), we don't want to manually open transactions panel. We just want to change the tab (in redeemRetryable for example)